docs/guides/react-native.md

```diff
@@ -1,6 +1,6 @@
 # Using Enzyme to Test Components in React Native
 
-As of React 0.18, React Native uses React as a dependency rather than a forked version of the library,
+As of React Native 0.18, React Native uses React as a dependency rather than a forked version of the library,
```

Comment:
I thought that sounded awkward as well, but since I was trying to be explicit about what we were talking about, I didn't want to completely remove it.

However, removing it completely still accomplishes the task of making sure it doesnt just say "React" so I agree with this.